config/missing-page-redirector.php

```diff
@@ -58,7 +58,7 @@
         /*
          * achievement tickets
          */
-        '/reportissue.php' => '/achievement/{i}/tickets/create',
+        '/reportissue.php' => '/achievement/{i}/report-issue',
```

Comment:
Was there an intention for us at one point to support an `/achievement/{id}/tickets` route?